src/config/configmanager.cpp

```diff
@@ -383,11 +383,17 @@ bool ConfigManager::reload() {
 	return result;
 }
 
+void ConfigManager::missingConfigWarning(const char* identifier) {
+	g_logger().warn(fmt::format("{}: Missing configuration for identifier: {}", __FUNCTION__, std::string(identifier)));
+}
+
```

Comment:
I think, creating a function it's not necessary. You can call the logger directly.